### ISSUE TITLE
Opening a file on a wrong page

### ISSUE THREAD

**Issue by GroupJavaJunior** [mozilla/pdf.js#12249]
[GED_2020-08-14_825.pdf](https://github.com/mozilla/pdf.js/files/5101040/GED_2020-08-14_825.pdf)

Configuration:
- Web browser and its version: Chrome 84.0.4147.125 (Official Build) (64-bit);
- Operating system and its version: Ubuntu 20.04;
- PDF.js version: 2.4.456;
- Is a browser extension: no.

Steps to reproduce the problem:
1. Open demo;
2. Open attached file.

What is the expected behavior?
I expected see the first page after opening.
![Screenshot from 2020-08-20 09-23-53](https://user-images.githubusercontent.com/28600497/90724255-ed3fba00-e2c6-11ea-8990-34f4201396fa.png)

What went wrong?
![vid-2020-08-20_09 27 06](https://user-images.githubusercontent.com/28600497/90724778-c1710400-e2c7-11ea-8998-66b58afb6ebc.gif)
That file always opens on the second page.
When I was investigating this issue, I saw that in the webViewerUpdateViewarea function, when scrolling the file with the mouse wheel up or down, the location.left property changes, while the location.top remains unchanged. For other files the location.top changes, the location.left one remains in place.

Link to a viewer:
https://mozilla.github.io/pdf.js/web/viewer.html or https://mozilla.github.io/pdf.js/es5/web/viewer.html

**Comment by THausherr** [comment]
Doesn't happen with me, W10, FF 79.0, using  https://mozilla.github.io/pdf.js/web/viewer.html

**Comment by GroupJavaJunior** [comment]
> Doesn't happen with me, W10, FF 79.0, using https://mozilla.github.io/pdf.js/web/viewer.html

Do you use attached file?
https://github.com/mozilla/pdf.js/files/5101040/GED_2020-08-14_825.pdf

**Comment by THausherr** [comment]
I used the one from the link on top, and just tested with the link you posted. Now FF 80.0.

**Comment by GroupJavaJunior** [comment]
Could you please try to open the document several times and scroll to the second page 2. Maybe the first time you open it, it was ok. For me it reproduced after second opening in FF 80.

**Comment by THausherr** [comment]
Doesn't happen with me. Maybe film this and post the video to youtube.

**Comment by GroupJavaJunior** [comment]
As I mentioned above. Please see if this works correctly:

> When I was investigating this issue, I saw that in the webViewerUpdateViewarea function, when scrolling the file with the mouse wheel up or down, the location.left property changes, while the location.top remains unchanged. For other files the location.top changes, the location.left one remains in place.

https://www.youtube.com/watch?v=TB_z7yz6zOw&feature=youtu.be

**Comment by GroupJavaJunior** [comment]
Hello everyone, is there any news about this ticket?

**Comment by GroupJavaJunior** [comment]
Hi, I investigated that wrong scroll is generated on the line:
`var boundingRect = [pageView.viewport.convertToViewportPoint(x, y), pageView.viewport.convertToViewportPoint(x + width, y + height)];` in viewer.js scrollPageIntoView function.
Could you please check why for this file it wrong?

**Comment by johnnyb8** [comment]
@makcormick We are also experiencing this. Can't reproduce on their demo viewer (https://mozilla.github.io/pdf.js/web/viewer.html) but our implementation has this issue. Confirmed with your file as well. Did you find a solution?

**Comment by makcormick** [comment]
@johnnyb8 No, I didn't find a solution, in my application in the pdf js viewer this still opens on the second page. I use pdf js for old browsers https://mozilla.github.io/pdf.js/legacy/web/viewer.html.

**Comment by DanielPower** [comment]
@makcormick Just to clarify, are you able to reproduce the issue in the demo viewer you linked to, or can you only reproduce in your application, which is using the legacy viewer?

We are also having the issue in our integrated pdf.js, but cannot reproduce on the demo viewer.

**Comment by makcormick** [comment]
@DanielPower I able to reproduce the issue in my application integrated with pdf js (stable 2.6 version). On the demo viewer I can't reproduce the issue.

**Comment by makcormick** [comment]
@DanielPower Hmm, strange, I had this error in the morning, now I cleared the cache and tried again, now it seems to open on the first page as expected. Perhaps the issu was resolved with the update of pdf js to version 2.6 for me
